src/uct/ib/rc/base/rc_iface.c

```diff
@@ -556,6 +559,44 @@ uct_rc_iface_init_max_rd_atomic(uct_rc_iface_t *iface,
     return UCS_OK;
 }
 
+static ucs_status_t
+uct_rc_iface_recv_mpool_init(uct_ib_iface_t *iface,
```

Comment:
avoid code duplication with uct_ib_iface_recv_mpool_init

Reply:
Can replace uct_ib_iface_recv_mpool_init with new uct_rc_iface_recv_mpool_init implementation?

Reply:
adds more params to uct_ib_iface_recv_mpool_init